### PR TITLE
Implement Ably Rule Resource with Zapier Target

### PR DESCRIPTION
**Pull request by graham-russell**
This PR includes the following items
- tfplugindocs generated documentation
- Ably Zapier integration rule resource with inbuilt source and target.
- Updates to provider and models to represent Ably Rule for Zapier target
- Playground tf templates updated
- Resource example updated for Zapier integration rule
- Acceptance testing configured

### CODE REVIEW

**Comment by graham-russell** [comment]
Note: Moving this to draft as I'll be rebasing off of Morgan's work on rules.

File: internal/provider/resource_ably_rule_zapier.go [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+// Get Rule Resource schema
+func (r resourceRuleZapierType) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return GetRuleSchema(
+		map[string]tfsdk.Attribute{
+			"url": {
+				Type:        types.StringType,
+				Required:    true,
+				Description: "The webhook URL that Ably will POST events to",
+			},
+			"headers": {
```
Comment (Morganamilo) [inline internal/provider/resource_ably_rule_zapier.go:27]: This should probably be pulled to a headers function.

**Review by Morganamilo — CHANGES_REQUESTED** [review]

**Review by kavalerov — CHANGES_REQUESTED** [review]
Can we please also add high level documentation line to the rule resource, similar to https://github.com/ably/terraform-provider-ably/blob/c5a80850748c871cfd3a50952903eebd7eacc1be/internal/provider/resource_ably_key.go#L83 ? 

The content for the line can be simple "The `ably-rule-zapier` allows you to create and manage Able integration rule for Zapier. Read more at https://ably.com/docs/general/webhooks/zapier"

**Review by Morganamilo — APPROVED** [review]

**Comment by graham-russell** [comment]
> Can we please also add high level documentation line to the rule resource, similar to
> 
> https://github.com/ably/terraform-provider-ably/blob/c5a80850748c871cfd3a50952903eebd7eacc1be/internal/provider/resource_ably_key.go#L83
> 
> ?
> The content for the line can be simple "The `ably-rule-zapier` allows you to create and manage Able integration rule for Zapier. Read more at https://ably.com/docs/general/webhooks/zapier"

This has been done now for all existing rule resources. (Kinesis/SQS/Lambda/Zapier)

**Comment by kavalerov** [comment]
Great, thanks!

**Review by kavalerov — APPROVED** [review]
LGTM, thanks

**Pull request merged**